radio/src/gui/480x272/radio_sdmanager.cpp

```diff
@@ -235,15 +236,15 @@ bool menuRadioSdManager(event_t _event)
           else if (isImageFileExtension(ext)) {
             TCHAR lfn[_MAX_LFN+1];
             f_getcwd(lfn, _MAX_LFN);
-            if (!READ_ONLY() && unsigned(LEN_FILE_EXTENSION+ext-line) <= sizeof(g_model.header.bitmap) && !strcmp(lfn, BITMAPS_PATH)) {
+            if (!READ_ONLY() && strlen(line) <= sizeof(g_model.header.bitmap) && !strcmp(lfn, BITMAPS_PATH)) {
```

Comment:
Yea... the old code seemed strange... subtracting the ext. length and then adding it back...  But now I see it's a quicker way to get the total length... clever.  How about we add just strlen(ext)?  

`unsigned(strlen(ext) + ext - line)`

Or do you see some other clever way to get the total length of `line` w/out assuming the length of file extension?